.changes/unreleased/tinymce-INT-3373-2025-10-02.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Chromium browsers would in certain situations scroll the editor unexpectedly when dragging content over the editor.
+time: 2025-10-02T15:42:09.368737+02:00
+custom:
+    Issue: INT-3373
```

Comment:
Should this really be a INT issue. The bug is not in the integrations but in the tinymce core. We only have one other ticket like this in the changelog. Can we log at TINY ticket and then remake this PR with that ticket?
We might need to check this with someone not sure if this is the procedure we want.